guillotina_rediscache/cache_strategy.py

```diff
@@ -143,15 +150,19 @@ def __init__(self, transaction, loop=None):
         '''
         push = {}
         for obj, pickled in self._stored_objects:
-            ob_key = '{}/{}'.format(obj.__parent__._p_oid, obj.__name__)
             val = {
                 'state': pickled,
                 'zoid': obj._p_oid,
                 'tid': obj._p_serial,
                 'id': obj.__name__
             }
-            # set new value in redis cache
-            await self.set(val, container=obj.__parent__, id=obj.__name__)
+            if obj.__of__:
+                ob_key = self.get_key(oid=obj.__of__, id=obj.__name__, variant='annotation')
```

Comment:
There can be multiple objects of , which is the obj.__name__ of the annotations

Reply:
Each annotation has a unique name. `annotation_object.__name__` is `default` for default behavior data. For example, it's set here: https://github.com/plone/guillotina/blob/master/guillotina/behaviors/instance.py#L20

Reply:
gresat !